src/octoprint/plugins/softwareupdate/__init__.py

```diff
@@ -113,12 +113,13 @@ def _get_configured_checks(self):
 						self._logger.exception("Error while retrieving update information from plugin {name}".format(**locals()))
 					else:
 						for key, default_config in hook_checks.items():
-							if key in effective_configs or key == "octoprint":
-								if key == name:
-									self._logger.warn("Software update hook {} provides check for itself but that was already registered by {} - overwriting that third party registration now!".format(name, check_providers.get(key, "unknown hook")))
-								else:
-									self._logger.warn("Software update hook {} tried to overwrite config for check {} but that was already configured elsewhere".format(name, key))
-									continue
+							#Disabled so the MrBeam Plugin can change this config
+							# if key in effective_configs or key == "octoprint":
+							# 	if key == name:
+							# 		self._logger.warn("Software update hook {} provides check for itself but that was already registered by {} - overwriting that third party registration now!".format(name, check_providers.get(key, "unknown hook")))
+							# 	else:
+							# 		self._logger.warn("Software update hook {} tried to overwrite config for check {} but that was already configured elsewhere".format(name, key))
+							# 		continue
```

Comment:
This is wrong. You are not meant to disable all the securities, only the restriction on modifying the `"octoprint"` key.

Reply:
You are right, will change